src/Console/Commands/CheckCommand.php

```diff
@@ -69,7 +70,30 @@ protected function execute(InputInterface $input, OutputInterface $output): int
     protected function configure(): void
     {
         $this->setDescription('Checks for misspellings in the given directory.')
-            ->addOption('config', 'c', InputOption::VALUE_OPTIONAL, 'The configuration file to use.', 'peck.json');
+            ->addOption(
+                'config',
+                'c',
+                InputArgument::OPTIONAL | InputOption::VALUE_REQUIRED,
+                'The configuration file to use.', 'peck.json'
+            );
+    }
+
+    /**
+     * If a user passes a config option to the command, attempt to load that file.
+     *
+     * If the file doesn't exist, throw an exception
+     *
+     * @throws InvalidOptionException
+     */
+    private function getConfigFile(InputInterface $input): void
+    {
+        $configurationPath = $input->getOption('config');
+
+        if (! is_string($configurationPath) || ! file_exists($configurationPath)) {
+            throw new InvalidOptionException('The config file you tried to load does not exist.');
```

Comment:
Maybe we could either catch the exception somewhere to display a pretty message to the user?
Or we could also just tell the User 'Invalid config file. Falling back to default!' 

Both are just suggestions and ideas! Feel free to tell me what you think.

Reply:
I'm going to re-look at all of the tests, Constantine. I didn't know about the `Process` part of Symfony that Nuno used the other day. So I'm going to look at using that instead.

Reply:
Yes, I never really used the Process Component either. I mean, if it works it works. 
Have a look and see what works best!

Reply:
That's OK - I've got it working with the CommandTester